src/utils/helpers.ts

```diff
@@ -0,0 +1,18 @@
+export const buildPath = (...args: string[]): string => {
+  return args
+    .map((part, i) => {
+      if (i === 0) {
+        return part.trim().replace(/[/]*$/g, '');
+      } else {
+        return part.trim().replace(/(^[/]*|[/]*$)/g, '');
+      }
+    })
+    .filter((x) => x.length)
+    .join('/');
+};
+
+export const buildRelativePath = (...args: string[]): string => {
+  const path = buildPath(...args);
+
+  return path.startsWith('/') ? path : `/${path}`;
+};
```

Comment:
Could you add a quick jsdoc comment for `buildPath` and `buildRelativePath`? Would be beneficial to give a brief explanation of what they do and why they are necessary right now. You could even use a condensed version of the comment you left previously regarding the many permutations of possible path values.